libs/vkd3d/command.c

```diff
@@ -2543,8 +2521,13 @@ static void d3d12_command_list_clear_attachment_pass(struct d3d12_command_list *
 
     stencil_attachment_info = attachment_info;
 
-    memset(&rendering_info, 0, sizeof(rendering_info));    
+    memset(&rendering_info, 0, sizeof(rendering_info));
     rendering_info.sType = VK_STRUCTURE_TYPE_RENDERING_INFO_KHR;
+    rendering_info.renderArea.offset.x = 0;
```

Comment:
This part seems like it should be folded into the previous commit.

Reply:
done.